apps/test-suite/publish.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -I nixpkgs=../../nix --packages expo-cli git yarn -i bash
+
+branch="$(git rev-parse --abbrev-ref HEAD)"
+channel=${1:-$branch}
+
+# Run yarn install in root, so linked dependencies are built
+pushd ../..
+for i in {1..3}; do ((i > 1)) && sleep 5; yarn install && break; done
+popd
+
+export EXPO_DEBUG=true
+
+# Both of these variables are necessary to publish with sdkVersion "UNVERSIONED"
+export EXPO_SKIP_MANIFEST_VALIDATION_TOKEN=true
+export EXPO_NO_DOCTOR=true
+
+expo login --username exponent_ci_bot --password "$EXPO_CLI_PASSWORD"
```

Comment:
FYI this password is already in git-crypt. Can you either remove it from git-crypt or from CircleCI so we have only one copy of it? (Slight preference for keeping the git-crypt copy so that we can run this script locally more easily and it should be as secure AFAICT.)

Reply:
Ah, forgot about those.